src/mount/sauna_client.cc

```diff
@@ -233,6 +233,7 @@ uint8_t get_session_flags() { return session_flags; }
 bool isMasterDisconnected() { return gIsDisconnectedFromMaster.load(); }
 
 #define LOCAL_USERS_THRESHOLD 0x30000
+#define IIS_USER 17
```

Comment:
Are we sure IIS is always using this hard coded `17` value. Consider making this a parameter that could be tweak on the  fly.

Reply:
I agree with @Antuan-AE.
It would be very nice to support updating this value at the configuration level.

Is it possible to define an option for the client?
For example:
`-o allowedUsers=17,123,456`